.circleci/config.yml

```diff
@@ -700,39 +700,39 @@ jobs:
       - checkout
       - install-bazel-yum:
           bazel-arch: arm64
-      - deploy-pip-release-unix
+#      - deploy-pip-release-unix
       - deploy-maven-jni-release-unix
-      - deploy-clib-release-unix
```

Comment:
It shouldn't be needed for the JNI work as everything is a part of the JNI library, right? Is it just for the standalone C library?

Reply:
(the C driver is not ready, I only touched the path to the outer drivers)

Reply:
Yes it doesn't have to be released for JNI to work, we just need the JNI to bundle the c library